docs/slider/readme.md

```diff
@@ -33,12 +33,8 @@ You can also add elements to it by using the tooltip (e.g. to display value) and
 | onDragStart | func                                  |              |            | Callback function that is fired when the slider has begun to move.<br>`function(event: object) : void`<br> ● event: MouseDown or TouchStart event targeting the slider. |
 | onDragStop  | func                                  |              |            | Callback function that is fired when the slide has stopped moving.<br>`function(event: object) : void`<br> ● event: MouseEnd or TouchEnd event targeting the slider. |
 | tooltip     | node                                  | null         |            | You can pass a tooltip component into this prop, and it will be displayed on hover near the handle of the Slider. |
-| error       | node                                  |              |            | NEED TO RESEARCH                         |
-| RTL         | bool                                  | FALSE        |            | NEED TO RESEARCH                         |
 
-### Code Examples
-
-#### **Example 1**
+### Code Example
 
 ```jsx
 //TODO: code guys - fix code example!
```

Comment:
Good question. @giladgrush Is the example broken? Or can I just get rid of the TODO comment?